util/travis_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+TRAVIS_COMMIT_MESSAGE="${TRAVIS_COMMIT_MESSAGE:-none}"
+TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE:-HEAD~1..HEAD}"
+
+# test force push
+#TRAVIS_COMMIT_RANGE="c287f1bfc5c8...81f62atc4c1d"
+
+NUM_IMPACTING_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE} | grep -Ev '^(docs/)' | wc -l)
```

Comment:
If you're doing a `-l` for wc that only counts lines. Couldn't we simplify this by just putting in a -c in grep, resulting in

```suggestion
NUM_IMPACTING_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE} | grep -Ecv '^(docs/)' )
```

Reply:
I lazily copied what `travis_build.sh` did, maybe as another PR we should change those too?

Reply:
Yeah, probably a good idea.